src/types/meets.ts

```diff
@@ -25,7 +25,7 @@ export type Race = {
   end_unix?: number;
   close?: string;
   close_unix?: number;
-  status: "Normal" | "Interim" | "Paying";
+  status: "Normal" | "Interim" | "Paying" | "Abandoned";
```

Comment:
Lets order this way...

```suggestion
  status: "Abandoned" | "Interim" | "Normal" | "Paying" ;
```

Reply:
will do